src/pods/configure-verbs/configure-verbs.component.tsx

```diff
@@ -15,30 +15,68 @@ interface Props {
   onCancel: () => void;
 }
 
-export const ConfigureVerbsComponent: React.FC<Props> = props => {
-  const { verbCollection, onSave, onCancel } = props;
-  const [temporalSelection, setTemporalSelection] = React.useState(
-    verbCollection
-  );
+const useTemporalSelectionAndSelected = (verbCollection: VerbEntity[]) => {
+  const [temporalSelection, setTemporalSelection] = React.useState<VerbEntity[]>([]);
   const [temporalSelected, setTemporalSelected] = React.useState(0);
 
   React.useEffect(() => {
     setTemporalSelection(verbCollection);
     setTemporalSelected(getSelectedNumber(verbCollection));
   }, [verbCollection]);
 
+  const increaseOrDecreaseTemporalSelected = (selected: boolean) => {
+    if (selected) {
+      setTemporalSelected(temporalSelected - 1);
+    } else {
+      setTemporalSelected(temporalSelected + 1);
+    }
+  };
+
+  const updateAfterMasterCheckboxChange = () => {
```

Comment:
ResetSelectionBasedOnRootSelectionCheckBox ?

Reply:
Ok, I will change the name of the method.